packages/frontend/core/src/components/pure/cmdk/main.tsx

```diff
@@ -44,6 +44,10 @@ const categoryToI18nKey: Record<CommandCategory, i18nKey> = {
   'editor:page': 'com.affine.cmdk.affine.category.editor.page',
 };
 
+export function removeDoubleQuotes(str?: string): string | undefined {
```

Comment:
Maybe move this function to `data` and escape the `command.value` directly in `pageToCommand`.